include/experimental/__p0009_bits/mdspan.hpp

```diff
@@ -311,7 +322,7 @@ class mdspan
   #endif // __cpp_lib_span
   #endif // MDSPAN_USE_PAREN_OPERATOR
 
-  MDSPAN_INLINE_FUNCTION constexpr size_t size() const noexcept {
+  MDSPAN_INLINE_FUNCTION constexpr size_type size() const noexcept {
```

Comment:
this returns size_type not size_t. Note that actually makes one of our old tests not work, I removed that thing.